spec/lib/graphql_rails/controller_spec.rb

```diff
@@ -374,7 +374,7 @@ class DummyMultipleBeforeActionsChildController < DummyMultipleBeforeActionsPare
 
           it 'adds error' do
             call
-            expect(context).to have_received(:add_error).with(SystemError.new('ups!'))
+            expect(context).to have_received(:add_error).with(kind_of(SystemError))
```

Comment:
Could you update this test or add new test where you check that `add_error` has been called with `SystemError` which contains data about raised error from [test setup]? 🙏 (https://github.com/samesystem/graphql_rails/blob/909ef0eb2215d67bb3dec92fab082e10c752ab23/spec/lib/graphql_rails/controller_spec.rb#L142)

Reply:
Done :)